spec/models/data_source_adapters/mysql2_adapter_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe DataSourceAdapters::Mysql2Adapter, type: :model do
```

Comment:
> __[RuboCop]__ Metrics/BlockLength: Block has too many lines. [28/25] ([link](https://sider.review/gh/repos/55787616/pulls/114#issue-3762142))

I don't like this rule 😊

Reply:
https://github.com/hogelog/dmemo/commit/64ac8ef7996bdc70de32d368933252042e9eae7c disabled